src/auth/interceptors/createRetryInterceptor.js

```diff
@@ -0,0 +1,63 @@
+import axios from 'axios';
+
+const createRetryInterceptor = (options = {}) => {
```

Comment:
1. I am not seeing a test for this, but this relates to point 2.
2. My brain is not functioning will enough to review this interceptor and ensure it is doing what you want. :)

Reply:
This is tested here https://github.com/edx/frontend-platform/pull/35/files#diff-872aa6abb07536f1f3aa4af84fe1985cR438 . Not sure if it is better to have a separate describe block.